### PR TITLE
Add exclude folder option + doc

### PR DESCRIPTION
**Pull request by heralight**
Add exclude folder option + doc

ideaExcludeFolders += ".idea_modules"

### CODE REVIEW

File: src/main/scala/org/sbtidea/SbtIdeaPlugin.scala [hunk 1]
```diff
@@ -93,8 +95,10 @@ object SbtIdeaPlugin extends Plugin {
 
     val projectInfo = IdeaProjectInfo(buildUnit.localBase, name.getOrElse("Unknown"), subProjects, ideaLibs ::: scalaLibs)
 
+    val excludeFolders = (ideaExcludeFolders in extracted.currentRef get buildStruct.data).getOrElse(Nil) :+ "target"
```
Comment (jozic) [inline src/main/scala/org/sbtidea/SbtIdeaPlugin.scala:98]: what about adding "target" to defaults, so it can be overridden?

Comment (heralight) [inline src/main/scala/org/sbtidea/SbtIdeaPlugin.scala:98]: ... Nil) :+ "target" 

was added to simplify configuration (in my case, "target" was excluded everytime ).
and yes, instead, we can adding "target" to defaults like:
ideaExcludeFolders <<= ideaExcludeFolders ?? "target" :: Nil;

As the leader prefers ...

Comment (mpeltonen) [inline src/main/scala/org/sbtidea/SbtIdeaPlugin.scala:98]: Well, I'm fine with how it currently is in pull request, at least initially, as the target folder was always excluded before this anyway. Let's move it to defaults later if someone comes up with a good reason to not always exclude target dir :)

**Comment by mpeltonen** [comment]
Merged, thanks!

**Pull request closed without merging**